python-package/lightgbm/sklearn.py

```diff
@@ -333,6 +334,12 @@ def fit(self, X, y,
         """
         evals_result = {}
         params = self.get_params()
+        # sklearn interface has another naming convention
+        params['seed'] = params.pop('seed', params.pop('random_state'))
+        params.pop('random_state', None)  # Avoid unknown parameter error from `train`
```

Comment:
@wxchan It seems that lines 339 and 341 are excess because `params.pop('random_state')` is called even if `seed` in `params`... Can you confirm it?